### PR TITLE
init_table

### PR DESCRIPTION
**Pull request by ilyapozh**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,7 +1,7 @@
 .DS_Store
 node_modules
 /dist
-
+/docker
```
Comment (tomfun) [inline .gitignore:4]: Better add `/docker/data`

File: my-vue3-app/src/components/ReportsTable.vue [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+import { Report } from '@/services/api';
+import { debounce } from 'lodash-es';
+
+type FetchParams = {
+  page: number;
+  pageSize: number;
+  filters: {
+    nick: {
+      value: string|undefined;
+      type: string;
```
Comment (tomfun) [inline my-vue3-app/src/components/ReportsTable.vue]: So we decided to change our API for the component

File: my-vue3-app/src/components/ReportsTable.vue [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+      fetchDebounced: debounce(
+        () => this.fetchReports(this.fetchParams),
+        150, {
+          maxWait: 5000,
+        },
+      ),
+      filters: {
+        nick: { value: undefined, matchMode: 'equals' },
+        title: { value: undefined, matchMode: 'equals' },
+      },
```
Comment (tomfun) [inline my-vue3-app/src/components/ReportsTable.vue]: use type `FetchParams` here

**Review by tomfun — CHANGES_REQUESTED** [review]

File: my-vue3-app/src/components/ReportsTable.vue (hunk 2 above)
Comment (ilyapozh) [reply my-vue3-app/src/components/ReportsTable.vue]: +

**Pull request merged**